staging/src/k8s.io/kubectl/pkg/cmd/apiresources/apiresources.go

```diff
@@ -111,6 +116,12 @@ func (o *APIResourceOptions) Validate() error {
 	if !supportedOutputTypes.Has(o.Output) {
 		return fmt.Errorf("--output %v is not available", o.Output)
 	}
+	supportedSortTypes := sets.NewString("", "name", "kind")
+	if len(o.SortBy) > 0 {
+		if !supportedSortTypes.Has(o.SortBy) {
+			return fmt.Errorf("--sort-by accepts only name or kind", o.Output)
```

Comment:
unnecessary `o.Output`:
```
fmt.Errorf("--sort-by accepts only name or kind")
```

Reply:
Thank you for the feedback! I updated my changes 👍